### PR TITLE
feat: Add support for 1.19

### PR DESCRIPTION
**Pull request by NotMyFault**
## Submitter Checklist

- [ ] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [ ] Ensure that the pull request title represents the desired changelog entry.
- [ ] New public fields and methods are annotated with `@since TODO`.
- [ ] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java [hunk 1]
```diff
@@ -461,13 +464,13 @@ public synchronized <T extends Future<T>> T call(IChunkSet set, Runnable finaliz
                             synchronized (super.sectionLocks[getSectionIndex]) {
                                 LevelChunkSection existingSection = levelChunkSections[getSectionIndex];
                                 if (createCopy && existingSection != null) {
-                                    copy.storeBiomes(getSectionIndex, existingSection.getBiomes().copy());
+                                    copy.storeBiomes(getSectionIndex, existingSection.getBiomes().recreate().copy());
```
Comment (dordsor21) [inline worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java:467]: Recreate does not seem to do the same thing as copy, and instead seems to recreate with no data

File: worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java [hunk 2]
```diff
@@ -489,7 +497,7 @@ public synchronized <T extends Future<T>> T call(IChunkSet set, Runnable finaliz
                                         }
                                     }
                                 } else {
-                                    PalettedContainer<Biome> biomeData = existingSection.getBiomes();
+                                    PalettedContainer<Holder<Biome>> biomeData = existingSection.getBiomes().recreate();
```
Comment (dordsor21) [inline worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java:500]: Is there a need to recreate the biome container? To me this just un-mutables it

File: worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java [hunk 3]
```diff
@@ -585,7 +604,7 @@ public synchronized <T extends Future<T>> T call(IChunkSet set, Runnable finaliz
                                 sectionLock.writeLock().unlock();
                             }
 
-                            PalettedContainer<Biome> biomeData = existingSection.getBiomes();
+                            PalettedContainer<Holder<Biome>> biomeData = existingSection.getBiomes().recreate();
```
Comment (dordsor21) [inline worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java:607]: More recreate?

File: worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java (hunk 1 above)
Comment (NotMyFault) [reply worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java:467]: Did this earlier when IJ refused to index paperweight to make it compile while browsing the jar, but apparently we can't call #copy() on here anymore, same like below?

**Review by NextdoorPsycho — APPROVED** [review]
This works, but you will have a block iterable issue. and you should use the block registry instead

**Pull request merged**